Cakefile

```diff
@@ -29,6 +29,15 @@ header = """
 # Used in folder names like `docs/v1`.
 majorVersion = parseInt CoffeeScript.VERSION.split('.')[0], 10
 
+# Patterns of names of test files which depend on currently absent features
+testFilesToSkip = [
+  /async/           unless try new Function 'async ()   => {}'
+  /async_iterators/ unless try new Function 'async () * => {}'
+  /exponent_ops/    unless try new Function 'x = 3; x **= 2 ** 2; return x === 81'
+].filter _.identity
+
+featuresPresentFor = (fileName) ->
```

Comment:
Can you please explain the name of this variable? Why create this function as a separate variable at all, rather than an inline argument to `filter` below (similar to the old code)?

Reply:
Maybe just put the function inline next to `filter`? Then it doesn’t need a name.